_common/README-check-version-env-vars.md

```diff
@@ -78,4 +78,4 @@
     ...
     ```
 
-Continue to **[Step 4: Setup Project and Run](./README-CS.md#step-4-setup-project-and-run)**
\ No newline at end of file
+Go back to continue with the rest of the demo sample.
```

Comment:
```suggestion
Go back to continue with the rest of the demo.
```

Reply:
Updated to use `example` in all the places.